configs/ote_custom_classification/efficientnet_b0/template.yaml

```diff
@@ -29,7 +29,7 @@ hyper_parameters:
     learning_parameters:
       batch_size:
         default_value: 32
-      max_num_epochs:
+      num_iters:
```

Comment:
Actually we plan to change `num_iters` to `max_num_epochs` in mmdetection, could you do that?
For deep-object-reid it should be `max_num_epochs` as well

Reply:
It need changes in mmdetection, mmsegmentation and all common test helpers files as well. need to consider it with Leonid.

Reply:
Anyway, on recent test jobs we have reproduced bug with `AssertionError: Wrong optimization type`, for POT step, need to reopen bug and postpone merge until it will be resolved.

Reply:
Refactored, restored 'max_num_epochs'.